src/video/emscripten/SDL_emscriptenevents.c

```diff
@@ -458,28 +458,25 @@ Emscripten_HandleTouch(int eventType, const EmscriptenTouchEvent *touchEvent, vo
             }
             SDL_SendTouch(deviceId, id, SDL_TRUE, x, y, 1.0f);
 
-            if (!preventDefault && SDL_GetEventState(SDL_FINGERDOWN) == SDL_ENABLE) {
-                preventDefault = 1;
-            }
+            // block browser's simulated mousedown
+            preventDefault = 1;
         } else if (eventType == EMSCRIPTEN_EVENT_TOUCHMOVE) {
             if ((window_data->finger_touching) && (window_data->first_finger == id)) {
                 SDL_SendMouseMotion(window_data->window, SDL_TOUCH_MOUSEID, 0, mx, my);
             }
             SDL_SendTouchMotion(deviceId, id, x, y, 1.0f);
 
-            if (!preventDefault && SDL_GetEventState(SDL_FINGERMOTION) == SDL_ENABLE) {
-                preventDefault = 1;
-            }
+            // no browser's simulated mousemove if we block mousedown
+            preventDefault = 0;
```

Comment:
Shouldn't need this (it's already 0)

Reply:
Hi!

As far as I can test, "wheel" events are handled independently (HandleWheel), and the browser does not simulate them on touch devices. We should be safe on that front :)

I added the "preventDefault = 0" line for clarity / symmetry.
I believe it is removed by trivial compiler optimization.
Do you want me to drop/comment it?